src/web/src/components/LanguageSelector.tsx

```diff
@@ -0,0 +1,53 @@
+import { createContext, ChangeEvent, useState } from 'react';
+import {
+  createStyles,
+  FormControl,
+  Grid,
+  InputLabel,
+  ListItemSecondaryAction,
+  MenuItem,
+  Select,
+} from '@material-ui/core';
+
+import changeLang from '../translations/i18n';
+
+export interface TranslationInterface {
+  currentLanguage: string;
+}
+
+export const translationInterface = createContext<TranslationInterface>({
+  currentLanguage: 'en',
+});
+
+const languages: Array<{ name: string; code: string }> = [
```

Comment:
Typescript can infer the type here for you.
```suggestion
const languages = [
```

Reply:
right - no difference.